### ISSUE TITLE
WALinuxAgent fails to start after update to latest version 2.7.3.0-1_ol001.el8

### ISSUE THREAD

**Issue by almalinux4all** [Azure/WALinuxAgent#2699]
Hello,

After updating WALinuxAgent to latest available version 2.7.3.0-1_ol001.el8 on a RHEL 8 VM, the waagent.service did not start correctly, and throws the following in the logs:

> Nov 05 18:37:24 test-rhel8-vm systemd[35838]: waagent.service: Failed to execute command: No such file or directory
Nov 05 18:37:24 test-rhel8-vm systemd[35838]: waagent.service: Failed at step EXEC spawning /usr/bin/python: No such file or directory

The environment is the following:
> [root@test-rhel8-vm ~]# waagent -version
WALinuxAgent-2.7.3.0 running on redhat 8.6
Python: 3.6.8
Goal state agent: 2.7.3.0

So in version 2.7.3.0-1_ol001.el8, the `waagent.service` file uses `/usr/bin/python` binary to start the WALinuxAgent, and this one does not exist in RHEL 8 by default. Looking into older versions of the WALinuxAgent package, like 2.5.0.2-1_ol001.el8, the  `waagent.service` file was configured to use `/usr/bin/python3` binary, which do exist in RHEL 8.

> [root@test-rhel8-vm ~]# ls -la /usr/bin/python*
lrwxrwxrwx. 1 root root    9 Feb 10  2022 /usr/bin/python2 -> python2.7
-rwxr-xr-x. 1 root root 8024 Feb 10  2022 /usr/bin/python2.7
lrwxrwxrwx. 1 root root   25 Mar  7  2022 /usr/bin/python3 -> /etc/alternatives/python3
lrwxrwxrwx. 1 root root   31 Aug 11  2021 /usr/bin/python3.6 -> /usr/libexec/platform-python3.6
lrwxrwxrwx. 1 root root   32 Aug 11  2021 /usr/bin/python3.6m -> /usr/libexec/platform-python3.6m

To workaround this issue, I have used `alternatives` to configure RHEL 8 with an `/usr/bin/python` binary as well, like below, but it would be great if this issue would be fixed in the package as well.

> [root@test-rhel8-vm ~]# alternatives --set python /usr/bin/python3
[root@test-rhel8-vm ~]#
[root@test-rhel8-vm ~]# ls -la /usr/bin/python*
lrwxrwxrwx. 1 root root   36 Nov  5 18:52 /usr/bin/python -> /etc/alternatives/unversioned-python
lrwxrwxrwx. 1 root root    9 Feb 10  2022 /usr/bin/python2 -> python2.7
-rwxr-xr-x. 1 root root 8024 Feb 10  2022 /usr/bin/python2.7
lrwxrwxrwx. 1 root root   25 Mar  7  2022 /usr/bin/python3 -> /etc/alternatives/python3
lrwxrwxrwx. 1 root root   31 Aug 11  2021 /usr/bin/python3.6 -> /usr/libexec/platform-python3.6
lrwxrwxrwx. 1 root root   32 Aug 11  2021 /usr/bin/python3.6m -> /usr/libexec/platform-python3.6m

Thank you,

**Comment by almalinux4all** [comment]
As a side note, I am using the latest WALinuxAgent RPM package from here: 
http://olcentgbl.trafficmanager.net/openlogic/8/openlogic/x86_64/RPMS/

**Comment by nagworld9** [comment]
Yes, we had a bug earlier and fixed in latest version. https://github.com/Azure/WALinuxAgent/pull/2532.

Please upgrade to latest version is 2.8.0.11
